db/schema.rb

```diff
@@ -36,6 +36,19 @@
 
   add_index "activities", ["template_id"], name: "index_activities_on_template_id", unique: true, using: :btree
 
+  create_table "bills", force: :cascade do |t|
+    t.string   "bill_id"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.